functions

```diff
@@ -846,7 +846,7 @@ try_enable_color() {
 
 install_modules() {
     local m moduledest=$BUILDROOT/lib/modules/$KERNELVERSION
-    local -a xz_comp gz_comp zst_comp
+    MODULES_DECOMPRESS="yes" && local -a xz_comp gz_comp zst_comp
```

Comment:
Combining these lines doesn't feel right to me.
```suggestion
    local -a xz_comp gz_comp zst_comp
    MODULES_DECOMPRESS="yes"
```

Reply:
Is it intentional that `install_modules` is forced to decompress? I don't quite see why we have the config option if we ignore it.

Reply:
Foxboron you are right, nl6720 proposal is not a fix.

Reply:
I reckon just removing the line is the correct approach here :)

Reply:
```suggestion
    local -a xz_comp gz_comp zst_comp
```